app/views/home/about.html.erb

```diff
@@ -107,6 +107,85 @@
+
+          <li>The <%= setting(:abbreviation) %> review process is clearly described on the journal’s Web site (https://joss.readthedocs.io/en/latest/reviewer_guidelines.html).</li>
+
+          <li>Judgments should be objective (https://joss.readthedocs.io/en/latest/reviewer_guidelines.html#guiding-principles).</li>
+
+          <li><%= setting(:abbreviation) %> reviewers should ideally have no conflict of interest. In practice, this is not always possible. If a reviewer has a conflict of interest, it must be declared and recorded, and the editors may choose to waive it if this is in the best interest of the review process (https://joss.readthedocs.io/en/latest/reviewer_guidelines.html#joss-conflict-of-interest-policy).</li>
+
+          <li>Reviewers should point out relevant published work which is not yet cited (https://joss.readthedocs.io/en/latest/review_criteria.html#an-important-note-about-novel-software-and-citations-of-relevant-work.</li>
+
+          <li><%= setting(:abbreviation) %> reviews are public and non-anonymous while in progress and post-review, as they take place via GitHub issues in a public repository.</li>
+        </ul>
+
+        <b>Publication ethics</b>
+        <ul>
+          <li>All authors are obliged to provide retractions or corrections of any mistakes of which they become aware.</li>
+          <li>Plagiarism (e.g., violation of another author's copyright), for both software and papers, is not allowed.</li>
+          <li>Self-plagiarism (repeated publication of the same work) is not allowed.</li>
+          <li>Author lists must be correct and complete. All listed authors must have made a contribution to the work, and all significant contributors should be included in the author list.</li>
+          <li>Reviews should be accurate and non-fraudulent. Examples of concerns are: Authors should not suggest reviewers who are not real people or have conflicts. Reviewers and editors must disclose conflicts. Bribes for authors, reviewers, editors are not permitted.</li>
+        </ul>
+
+        <b>Copyright and Access</b>
+        <ul>
+          <li><%= setting(:abbreviation) %> copyright and licensing information is clearly described on the journal’s web site (<%= setting(:url) %>/about#content_license).</li>
+          <li>The journal and all individual articles are freely available to all readers.</li>
+        </ul>
+
+        <b>Archiving</b>
+        <ul>
+          <li><%= setting(:abbreviation) %> articles, metadata, and reviews are archived with <a href="https://www.portico.org/">Portico</a>.</li>
+        </ul>
+
+        <b>Ownership and management</b>
+        <ul>
+          <li>Information about the ownership and/or management of a journal shall be clearly indicated on the journal’s web site. (http://www.theoj.org)</li>
```

Comment:
@danielskatz - do you think what we have under 'Ownership & Governance' on http://www.theoj.org is sufficient here?

Reply:
maybe change "Open Journals and each of its journals are volunteer-led" to "Open Journals and each of its journals are community-organized and volunteer-led"